src/Components/Cards/QuestionCard/QuestionCard.css

```diff
@@ -0,0 +1,222 @@
+.question-card,
+.question,
+.question-details,
+.quiz-detail,
+.quiz-total-time,
+.quiz-high-score,
+.quiz-progress {
+  display: flex;
+}
+
```

Comment:
Suggestion: You could directly provide flex property to the html tags using flex utility class.